source/Glimpse.Core/Framework/GlimpseConfiguration.cs

```diff
@@ -704,5 +704,11 @@ public static IMessageBroker GetConfiguredMessageBroker()
         {
             return messageBroker;
         }
+
+        [Obsolete("HACK: To support TraceListener with TraceSource via web.config")]
```

Comment:
maybe change the message to something like _HACK: To prevent unnecessary wrapping of SQL connections, commands, ..._ ?